bears/coffee_script/CoffeeLintBear.py

```diff
@@ -64,7 +64,7 @@ def generate_config(filename, file,
                         allow_this_statements: bool=True,
                         allow_increment: bool=True,
                         allow_empty_parameter_list: bool=True,
-                        no_empty_functions: bool=False,
+                        allow_empty_functions: bool=True,
```

Comment:
can you make another commit changing the default to False? I think we should warn here. FWIW we should go for sane defaults rather than just taking the linters defaults.